src/DMCHandler.h

```diff
@@ -3,11 +3,52 @@
 
 #include <iostream>
 #include <primitives/block.h>
+#include "rpc/client.h"
+
+class DFITx {
+    private:
+    char* from; // TODO: get correct DMC Address type
+    char* to; // TODO: get correct DFI address type
+    int64_t amount;
+    char* signature;
+
+public:
+    DFITx(char* from, char* to, int64_t amount, char* signature){
+        this->from = from;
+        this->to = to;
+        this->amount = amount;
+        this->signature = signature;
+    }
+};
+
+class DMCTx {
+private:
+    char* from; // TODO: get correct DFI address type
+    char* to; // TODO: get correct DMC Address type
+    int64_t amount;
+    char* signature;
+
+public:
+    DMCTx(char* from, char* to, int64_t amount, char* signature){
+        this->from = from;
+        this->to = to;
+        this->amount = amount;
+        this->signature = signature;
+    }
+};
+
+struct EncodedDMCBlock {
+    std::vector<unsigned char> blockdata;
+} EncodedDMCBlock;
 
 // WIP
 class DMCHandler {
+private:
+    rpc::client DMCNode;
```

Comment:
make this injectable. you can either pass it in the constructor or another method you prefer.
Then we can use a mock object of `DMCNode` to isolate from the DMC side and write more functional tests of its own on the native side.